docs/how_to_run.md

```diff
@@ -0,0 +1,9 @@
+## Scenarios
+
+### [Perception Pipeline Benchmark](scenarios/perception_pipeline_benchmark.md)
+
+This benchmark measures the elapsed time by which the determined path is sent for the robot to follow. This benchmark calculates `elapsed time`, `success_number` and `failure_number`. `elapsed_time` is used for measuring how much time the pipeline takes. `success_number` is used for presenting successfully plannings and `failure_number` is used for presenting failed plannings.
+
+Firstly, `node` and `move_group_interface`in SetUp are created before each benchmark. `poses` inside `nav_msgs/msg/Path` is sent one by one to plan trajectory for robot. If planning is failed, only `failure_rate` is increased. If planning is successful, the trajectory_plan which move_group_server plan is sent via `move_group_interface` to start the execution of this planned trajectory. Then `success_number` is increased.
```

Comment:
Does the selected middleware have any impact on the success / failure of the planning request? I'd assume it shouldn't and in that case these numbers might be not super relevant for the benchmark. What do you think?

Reply:
You're right. But we are working on experimental middleware as well. I'm trying to follow each updates in rmw_zenoh. It is possible that forward updates on rmw_zenoh break the benchmark pipeline. rmw_zenoh had problem for multithreaded executor before(It seems it's fixed now but there is still no guarantee that rmw_zenoh will not break our benchmark) and this bug sometimes broke our scenario. (communication between action_client and action_server). To sum up, i want not to remove this part because of above reasons. But if you think your idea is still better than mine, i can change this part.